soccer/KickEvaluator.hpp

```diff
@@ -80,11 +79,10 @@ class KickEvaluator {
      * @return F(X), F'(X)
      */
     static std::tuple<float, float> eval_calculation(
-        float x, const float kmean, const float kstdev,
```

Comment:
ah yeah you're right. In `readability-avoid-const-params-in-decls`, it only checks for `const` in declarations because they are superfluous, but they're not superfluous in definitions. I think for consistency we should also remove them in definitions though